xml/System.IO/FileInfo.xml

```diff
@@ -2506,7 +2506,7 @@ The following example demonstrates moving a file to a different location and ren
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>Returns the path as a string. Use the <see cref="P:System.IO.FileInfo.Name" /> property for the full path.</summary>
+        <summary>Returns the path as a string.</summary>
```

Comment:
```suggestion
        <summary>Returns the original path that was passed to the <see cref="T:System.IO.FileInfo" /> constructor. Use the <see cref="P:System.IO.FileInfo.FullName" /> or <see cref="P:System.IO.FileInfo.Name" /> property for the full path or file name.</summary>
```

This is from the [source code](https://github.com/dotnet/runtime/blob/dd980646055235a60cb709848478e128cbeabf38/src/libraries/System.Private.CoreLib/src/System/IO/FileSystemInfo.cs#L183C52-L183C60): 
> Returns the original path. Use FullName or Name properties for the full path or file/directory name.

And from DirectoryInfo:
> Returns the original path that was passed to the [DirectoryInfo](https://learn.microsoft.com/en-us/dotnet/api/system.io.directoryinfo?view=net-6.0) constructor. Use the [FullName](https://learn.microsoft.com/en-us/dotnet/api/system.io.directoryinfo.fullname?view=net-6.0#system-io-directoryinfo-fullname) or [Name](https://learn.microsoft.com/en-us/dotnet/api/system.io.directoryinfo.name?view=net-6.0#system-io-directoryinfo-name) properties for the full path or file/directory name instead of this method.

Reply:
This caused a "Cross reference not found: 'System.IO.FileInfo.FullName'" warning but it looks fine to me.

If you see `DirectoryInfo.ToString()` it uses a similar cross-ref https://github.com/dotnet/dotnet-api-docs/blob/6c1d2ade568ef7fc34766354dd5f0d2336499126/xml/System.IO/DirectoryInfo.xml#L3878

cc @gewarren @carlossanlop

Reply:
`FullName` is not a property of `FileInfo` or `DirectoryInfo`, [it's a property of the parent FileSystemInfo](https://learn.microsoft.com/en-us/dotnet/api/system.io.filesysteminfo.fullname?view=net-7.0) and it's not overridden by the children. That's why it's giving a warning.

You could replace it with `<c>System.IO.DirectoryInfo.FullName</c>`, or if you prefer to preserve a valid hyperlink, replace it with `<see cref="P:System.IO.FileSystemInfo.FullName"/>`.

Reply:
Why does it not warn on DirectoryInfo then?

Reply:
Wow, this is weird. I'm not actually seeing this text at all; I can only see the [FileSystemInfo.ToSTring()](https://learn.microsoft.com/en-us/dotnet/api/system.io.filesysteminfo.tostring?view=net-7.0#system-io-filesysteminfo-tostring) text. I'll log a bug.